### PR TITLE
feat(cloud): Introduce image quota output

### PR DESCRIPTION
**Pull request by craciunoiuc**
### Prerequisite checklist

  - [ ] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
  - [ ] Tested your changes against relevant architectures and platforms;
  - [ ] Ran `make fmt` on your commit series before opening this PR;
  - [ ] Updated relevant documentation.

### CODE REVIEW

**Comment by nderjung** [comment]
```
panic: strings: negative Repeat count

goroutine 1 [running]:
strings.Repeat({0x10446b9f4?, 0x10446dd39?}, 0x60?)
        /Users/nderjung/go/src/strings/strings.go:549 +0x2e0
kraftkit.sh/internal/cli/kraft/cloud/utils.printBar(0x14000468ec7, 0x14000736f30?, 0x1400063b700?)
        /Users/nderjung/Workspace/unikraft/kraftkit-pr-1620/internal/cli/kraft/cloud/utils/print.go:629 +0xb8
kraftkit.sh/internal/cli/kraft/cloud/utils.PrintQuotas({0x104ead358, 0x14000736f30}, {{0x14000320cc0, 0x1b}, {0x14000320cdc, 0x20}, {0x1044884da, 0x11}, 0x1}, {0x104b2fc93, ...}, ...)
        /Users/nderjung/Workspace/unikraft/kraftkit-pr-1620/internal/cli/kraft/cloud/utils/print.go:735 +0x2d84
kraftkit.sh/internal/cli/kraft/cloud/quotas.(*QuotasOptions).Run(0x140003b40c0, {0x104ead358, 0x14000736f30}, {0x0?, 0x0?, 0x0?})
        /Users/nderjung/Workspace/unikraft/kraftkit-pr-1620/internal/cli/kraft/cloud/quotas/quotas.go:97 +0x3fc
kraftkit.sh/cmdfactory.New.func1(0x140002dd808?, {0x10628ee28?, 0x0?, 0x0?})
        /Users/nderjung/Workspace/unikraft/kraftkit-pr-1620/cmdfactory/builder.go:371 +0x4c
kraftkit.sh/cmdfactory.AttributeFlags.bind.func3(0x140002dd808, {0x10628ee28, 0x0, 0x0})
        /Users/nderjung/Workspace/unikraft/kraftkit-pr-1620/cmdfactory/builder.go:553 +0x14c
github.com/spf13/cobra.(*Command).execute(0x140002dd808, {0x10628ee28, 0x0, 0x0})
        /Users/nderjung/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:983 +0x840
github.com/spf13/cobra.(*Command).ExecuteC(0x1400002e908)
        /Users/nderjung/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/nderjung/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        /Users/nderjung/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1032
kraftkit.sh/cmdfactory.Main({0x104ead358, 0x14000736f30}, 0x1400002e908)
        /Users/nderjung/Workspace/unikraft/kraftkit-pr-1620/cmdfactory/builder.go:183 +0x6c
kraftkit.sh/internal/cli/kraft.Main({0x16d043030?, 0x102dffe40?, 0x10613abe8?})
        /Users/nderjung/Workspace/unikraft/kraftkit-pr-1620/internal/cli/kraft/kraft.go:204 +0x884
main.main()
        /Users/nderjung/Workspace/unikraft/kraftkit-pr-1620/cmd/kraft/main.go:19 +0x94
```

**Comment by craciunoiuc** [comment]
strange, probably out of date now, will update

File: internal/cli/kraft/cloud/utils/print.go [hunk 1]
```diff
@@ -664,6 +665,15 @@ func PrintQuotas(ctx context.Context, auth config.AuthConfig, format string, res
 		table.AddField("", nil)
 	}
 
+	if imageResp == nil || imageResp.Used == -1 {
```
Comment (nderjung) [inline internal/cli/kraft/cloud/utils/print.go]: I think this is meant to be `imageResp != nil && imageResp.Hard > 0` ?

File: internal/cli/kraft/cloud/utils/print.go [hunk 2]
```diff
@@ -721,6 +731,24 @@ func PrintQuotas(ctx context.Context, auth config.AuthConfig, format string, res
 		table.AddField("", nil)
 	}
 
+	// IMAGE STORAGE
+	if imageResp == nil || imageResp.Used == -1 {
```
Comment (nderjung) [inline internal/cli/kraft/cloud/utils/print.go]: Same here

**Review by nderjung — APPROVED** [review]
Thanks!

**Pull request merged**